### PR TITLE
Update Prometheus Cluster Monitoring description in the 3.11 release notes

### PR DESCRIPTION
**Pull request by sichvoge**
@adellape I submitted a few edits to the release notes.

### CODE REVIEW

File: release_notes/ocp_3_11_release_notes.adoc [hunk 1]
```diff
@@ -427,14 +427,13 @@ Cluster Monitoring Operator] for capacity planning details.
...
 ==== Prometheus Cluster Monitoring
 
-Prometheus cluster monitoring is now fully supported in {product-title}.
+Prometheus cluster monitoring is now fully supported in {product-title} and deployed by default into an {product-title} cluster.
 
-You can deploy Prometheus on an {product-title} cluster, collect Kubernetes and
-infrastructure metrics, and get alerts. You can see and query metrics and alerts
-on the Prometheus web dashboard.
+* Query and plot cluster metrics collected by Prometheus.
+* Receive notifications from pre-packaged alerts, enabling owners to take correctiveactions and start troubleshooting problems.
```
Comment (ahardin-rh) [inline release_notes/ocp_3_11_release_notes.adoc]: corrective actions

File: release_notes/ocp_3_11_release_notes.adoc [hunk 2]
```diff
@@ -427,14 +427,13 @@ Cluster Monitoring Operator] for capacity planning details.
...
+* View pre-packaged Grafana dashboards for etcd, cluster state, and many other aspectsof cluster health.
```
Comment (ahardin-rh) [inline release_notes/ocp_3_11_release_notes.adoc]: aspect of

File: release_notes/ocp_3_11_release_notes.adoc [hunk 3]
```diff
@@ -427,14 +427,13 @@ Cluster Monitoring Operator] for capacity planning details.
...
 
-See xref:../install_config/cluster_metrics.adoc#openshift-prometheus[Prometheus
-on OpenShift] for more information.
+See xref:../install_config/prometheus_cluster_monitoring.adoc[Configuring Prometheus Cluster Monitoring] for more information.
```
Comment (ahardin-rh) [inline release_notes/ocp_3_11_release_notes.adoc]: This file does not exist, which is causing the build to fail

Comment (sichvoge) [reply release_notes/ocp_3_11_release_notes.adoc]: Yeah, it's not yet merged and currently in review. What are we doing in this case?

Comment (ahardin-rh) [reply release_notes/ocp_3_11_release_notes.adoc]: @rh-max @vikram-redhat Any update there? Thanks!

Comment (vikram-redhat) [reply release_notes/ocp_3_11_release_notes.adoc]: @sichvoge - The link is missing the anchor to point to. Besides that, we will need to wait to merge this till @rh-max is ready to merge his.

Comment (sichvoge) [reply release_notes/ocp_3_11_release_notes.adoc]: @rh-max What anchor do you want me to add?

Comment (rh-max) [reply release_notes/ocp_3_11_release_notes.adoc]: @ahardin-rh @sichvoge Use `xref::../install_config/prometheus_cluster_monitoring.adoc#prometheus-cluster-monitoring` please.

Comment (sichvoge) [reply release_notes/ocp_3_11_release_notes.adoc]: ok added.

**Comment by vikram-redhat** [comment]
@tmorriso-rh - this can be merged and CP'ed once #11393 merges.

**Comment by tmorriso-rh** [comment]
> @tmorriso-rh - this can be merged and CP'ed once #11393 merges.

Vikram, https://github.com/openshift/openshift-docs/pull/11393 remains Open at this time.

**Pull request merged**